test/swarm.js

```diff
@@ -554,11 +571,6 @@ test('multiple discovery sessions with different opts', async (t) => {
 
   await discovery1.destroy() // should not prevent server connections
   await discovery2.flushed()
```

Comment:
There's no need for `await discovery2.flushed()` here (so neither for storing that var)
(fwiw, there was no need before this PR either)